tools/ansible/roles/dockerfile/files/launch_awx_rsyslog.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+if [ `id -u` -ge 500 ]; then
+    echo "awx:x:`id -u`:`id -g`:,,,:/var/lib/awx:/bin/bash" >> /tmp/passwd
+    cat /tmp/passwd > /etc/passwd
+    rm /tmp/passwd
+fi
+
+if [ -n "${AWX_KUBE_DEVEL}" ]; then
+    pushd /awx_devel
+    make awx-link
+    popd
+
+    export SDB_NOTIFY_HOST=$MY_POD_IP
+fi
+
+set -e
+
+wait-for-migrations
+
+# This file will be re-written when the dispatcher calls reconfigure_rsyslog(),
+# but it needs to exist when supervisor initially starts rsyslog to prevent the
+# container from crashing. This was the most minimal config I could get working.
+cat << EOF > /var/lib/awx/rsyslog/rsyslog.conf
+action(type="omfile" file="/dev/null")
+EOF
```

Comment:
This seems okay to me still, after checking [Jessica's PR](
https://github.com/ansible/awx/pull/13222/files#diff-017d6c505750589da91821bb9b0e8319dca2450aba649dd55c921ff20dcb6b3aR24-R25). When the configurer starts up, it'll do an initial `reconfigure_rsyslog()` before it starts blocking and waiting to be told to reconfigure it in the future, so this minimal config shouldn't last very long. :+1: